modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -359,6 +411,21 @@ func (t TLSConfig) MakeTLSClientConfig(ctx caddy.Context) (*tls.Config, error) {
 	return cfg, nil
 }
 
+type UpstreamResolver struct {
+	// Specifies the addresses of the [DNS resolvers of upstram addresses resolvers]
+	// (/docs/json/apps/http/servers/routes/handle/reverse_proxy/transport/http/resolver/addresses).
```

Comment:
```suggestion
	// Specifies the addresses of the [DNS resolvers of upstream addresses resolvers](/docs/json/apps/http/servers/routes/handle/reverse_proxy/transport/http/resolver/addresses).
```

Can we clarify how this is different from Addresses, and why one/either is required?